pkgs/top-level/all-packages.nix

```diff
@@ -20115,6 +20115,8 @@ with pkgs;
 
   wiredtiger = callPackage ../development/libraries/wiredtiger { };
 
+  wlr-protocols = callPackage ../development/libraries/wlroots/protocols.nix { };
```

Comment:
The package definition should live in an directory that has the same name as the attribute name (`wlr-protocols`). Also I'm not sold on putting it into the wlroots directory. Is it really that integral to wlroots that we can't put it into `pkgs/development/libraries/wlr-protocols/default.nix`?

Reply:
My intent was to stay consistent to the layout for wayland-protocols. I can move it if you prefer?

Reply:
Ah I see, didn't know about that case. Makes sense!